index.html

```diff
@@ -1,75 +1,108 @@
+                        Blog
+                    </a>
+                    <a href="#" class="navLink">
+                        Contact
+                    </a>
+                </nav>
+            </div>
+            <div class="container">
+                <img src="img/jumbo.jpg">
+            </div>
+        </header>
         <section class="top-content">
             <div class="text-container">
                 <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt
+                    efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas
+                    tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
             <div class="text-container">
                 <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt
+                    efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas
+                    tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
         </section>
-        
+
         <section class="middle-content">
-        
+
             <h2>Why Did It Have To Be Boxes...</h2>
-            
+
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" id="box1">Box 1</div>
+                <div class="box" id="box2">Box 2</div>
+                <div class="box" id="box3">Box 3</div>
+                <div class="box" id="box4">Box 4</div>
+                <div class="box" id="box5">Box 5</div>
+                <div class="box" id="box6">Box 6</div>
+                <div class="box" id="box7">Box 7</div>
+                <div class="box" id="box8">Box 8</div>
+                <div class="box" id="box9">Box 9</div>
+                <div class="box" id="box10">Box 10</div>
             </div>
```

Comment:
The :nth-child css selector will allow you to be able to do the same without having to add additional ids or css -> https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child